packages/mock-server/src/utils/handleAuthentication.ts

```
@@ -0,0 +1,82 @@
+
+      for (const securityRequirement of operationSecuritySchemes) {
+        for (const [schemeName] of Object.entries(securityRequirement)) {
+          const scheme = schema?.components?.securitySchemes?.[schemeName]
+
+          if (scheme) {
+            switch (scheme.type) {
+              case 'http':
+                if (scheme.scheme === 'basic') {
+                  const authHeader = c.req.header('Authorization')
+
+                  if (authHeader && authHeader.startsWith('Basic ')) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)